arraycontext/container/traversal.py

```diff
@@ -265,4 +273,34 @@ def thaw(ary, actx):
 
 # }}}
 
+
+# {{{ numpy conversion
+
+def from_numpy(actx, ary):
```

Comment:
In case this gets in, `from_numpy(actx, ary)` or `from_numpy(ary, actx)`?

Reply:
Yes, let's go for consistency with `thaw` and `freeze`.

Reply:
Updated in bd76184.